src/appleDeviceUtils/appleDeviceUtils.test.js

```diff
@@ -0,0 +1,40 @@
+const {wkSendAndWait} = require('./AppleDeviceUtils')
+
+const ddgGlobals = window.navigator.ddgGlobals
+
+const webkitMock = jest.fn(async (data) => {
```

Comment:
Basic test used to make sure things are working as expected while the native layer is yet to be adapted. This will be expanded later on to cover more cases.